pkg/converter/v2/converter.go

```diff
@@ -140,6 +135,20 @@ func (c *converter) updateServers() error {
 			delete(server, "schemeVersion")
 		}
 	}
+
+	var mappedServers = make(map[string]interface{})
+
+	for index, item := range servers {
```

Comment:
I don't think there's need to document this behaviour, creating docs just for this is kinda overkill, and it does not fit into readme anywhere 😭. And it's not something that user can change, if they are using converter then they have to accept that many fields used to be arrays and are now objects, and keys have to autogenerated

Reply:
Only knowledge that an user would gain from those docs that what used to be array is now object, and keys go like: default, server1, server2

which makes me think that it makes more sense for it to go like: server1, server2, server3..., so that it looks autogenerated and people would know it

Reply:
But I was following JS converter on this matter

Reply:
@fmvilas What do you think about that?

Reply:
I think making it look like autogenerated when there's more than one is the best solution here. Honestly, I don't think that matters too much, people can always change it afterward. This tool is just to help with the migration process.